src/components/claim/Detail.vue

```diff
@@ -257,12 +257,12 @@
     }
 
     .card-title {
-      color: #32A4BE;
```

Comment:
Could we use css variable?

Reply:
it would be nice, but it requires migrating project to scss, doesn't it?

Reply:
I haven't used it myself yet, but I think there's an opportunity in CSS
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
https://caniuse.com/#feat=css-variables 92% of the support